### DISCUSSION TITLE
how to debugging in vscode

### DISCUSSION THREAD

**Discussion by zzjjbbaa** [wasm-bindgen/wasm-bindgen#4050] [Q&A]
### Summary

A clear and concise summary of your question.

### Additional Details

Provide any additional details here.

**Comment by daxpedda** [comment]
Well, despite the lack of any basic information:
see #3698 and recently just answered https://github.com/rustwasm/wasm-bindgen/issues/1289#issuecomment-2267543019, this should actually *just* work in VSCode.

Any other questions should be asked in a VSCode context instead.
